source/administration/migrating.rst

```diff
@@ -26,4 +26,70 @@ The following instructions migrate Mattermost from one server to another by back
 Migrating from Slack
 ====================
 
-You can import channels and users from Slack into Mattermost from Team Settings > Import. Please review our documentation on `Slack Import <https://docs.mattermost.com/help/settings/team-settings.html#import-from-slack-beta>`_ for more details.
+.. note:: As a proprietary SaaS service, Slack is able to change its export format quickly and without notice. If you encounter issues not mentioned in the documentation below, please alert the product team by `filing an issue <https://github.com/mattermost/platform/issues>`_.
+
+The Slack Import feature in Mattermost is in "Beta" and focused on supporting migration of teams of less than 100 registered users.
+
+This feature can be accessed through the `Mattermost Web App </administration/migrating.html#migrating-from-slack-using-the-mattermost-web-app>`_ or using the `CLI </administration/migrating.html#migrating-from-slack-using-the-mattermost-cli>`_.
+
+.. warning:: **It is highly recommended that you test Slack import before applying it to an instance intended for production.**
+
+   If you use Docker, you can spin up a test instance in one line (``docker run --name mattermost-dev -d --publish 8065:80 mattermost/platform``). If you don't use Docker, there are `step-by-step instructions <http://docs.mattermost.com/install/docker-local-machine.html>`_ to install Mattermost in preview mode in less than 5 minutes.
+
+Supported Features
+++++++++++++++++++
+
+The following key features are supported out of the box when importing from Slack:
+
+* User accounts with an email address set.
+
+* Public channels and the text messages posted in them by users.
+
+* Channel topic and purpose.
+
+* Automatically adding imported users to their channels.
```

Comment:
Reword to make the phrasing more consistent with previous bullet points, which start with a noun-adjective pair, e.g. `public channels`, `channel topic`

Perhaps `Imported users added automatically to their channels`

Only 1/5 here though,